### PR TITLE
feat(tooltip): add delay before show text [KMAPS-523]

### PR DESCRIPTION
**Pull request by nelsonmestevao**

### CODE REVIEW

File: packages/react-ui-components/src/stories/Components/TreeDropdown.stories.tsx [hunk 1]
```diff
@@ -30,7 +30,17 @@ const TreeDropdownTemplate: ComponentStory<typeof KvTreeDropdown> = args => {
...
 
-	return <KvTreeDropdown selectedNode={selectedNode} expandedNodes={expandedNodes} onNodeClick={handleItemClick} onNodeToggleExpand={handleToggleExpand} {...args} />;
+	return (
+		<KvTreeDropdown
+			selectedNode={selectedNode}
+			expandedNodes={expandedNodes}
+			onNodeClick={handleItemClick}
+			onNodeToggleExpand={handleToggleExpand}
+			showTooltip
+			tooltipDelay={1000}
```
Comment (djcouto) [inline packages/react-ui-components/src/stories/Components/TreeDropdown.stories.tsx:40]: Please move this to a config file. Also this should be a configurable prop.

File: packages/react-ui-components/src/stories/Components/TreeDropdown.stories.tsx [hunk 2]
```diff
@@ -30,7 +30,17 @@ const TreeDropdownTemplate: ComponentStory<typeof KvTreeDropdown> = args => {
...
 	};
 
-	return <KvTreeDropdown selectedNode={selectedNode} expandedNodes={expandedNodes} onNodeClick={handleItemClick} onNodeToggleExpand={handleToggleExpand} {...args} />;
+	return (
+		<KvTreeDropdown
+			selectedNode={selectedNode}
+			expandedNodes={expandedNodes}
+			onNodeClick={handleItemClick}
+			onNodeToggleExpand={handleToggleExpand}
+			showTooltip
```
Comment (djcouto) [inline packages/react-ui-components/src/stories/Components/TreeDropdown.stories.tsx:39]: This should also be a configurable prop.

**Review by djcouto — CHANGES_REQUESTED** [review]
Nice, just a few comments 💪

**Comment by djcouto** [comment]
Sorry, my comments didn't make much sense 😅

**Review by djcouto — APPROVED** [review]

**Review by miguelpinto98 — APPROVED** [review]
👍  Good preventing the previous behaviour, and nice improvement for we want now.

**Pull request merged**